glean-core/ffi/Cargo.toml

```diff
@@ -46,3 +46,6 @@ oslog = { version = "0.0.3", default-features = false, features = ["logger"] }
 
 [target.'cfg(not(target_os = "android"))'.dependencies]
 env_logger = { version = "0.7.1", default-features = false, features = ["termcolor", "atty", "humantime"] }
+
+[target.'cfg(unix)'.dependencies]
+libc = "0.2.82"
```

Comment:
Why do we need this?

Reply:
We use `dlsym` from `libc` to look up symbols. That works on Unix platforms (macOS, Linux), I'm not sure it would actually work on Windows the same way. Therefore we only need this dependency on unix platforms.

I'll file a followup to check Windows (it's less important because this feature is mainly to support Android right now)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1687957